core/src/main/java/org/apache/accumulo/core/clientImpl/MasterClient.java

```diff
@@ -103,17 +103,8 @@ public static void close(MasterClientService.Iface iface) {
         sleepUninterruptibly(100, TimeUnit.MILLISECONDS);
       } catch (ThriftSecurityException e) {
         throw new AccumuloSecurityException(e.user, e.code, e);
-      } catch (AccumuloException e) {
+      } catch (AccumuloException | ThriftTableOperationException e) {
         throw e;
-      } catch (ThriftTableOperationException e) {
```

Comment:
The intent here was to avoid leaking RPC types. Are we sure we want to throw the Thrift exception type outside of this method now? I think I'd prefer we didn't. I like the RPC code to be confined.

Reply:
OK.  I am not sure how to get rid of the Table wrapping Namespace craziness then.  If we make the generic method throw both, then the executeNamespace method will also throw a TableNFE and the excuteTable will also throw NamespaceNFE.  This adds more funkiness to the calling methods.